sonic_api.yaml

```diff
@@ -1255,6 +1255,282 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 #----------------------------------------------
+# Setup Global BGP process
+#----------------------------------------------
+  '/config/bgp/{asn}/{router_id}':
+    put:
+      summary: Setup Global BGP configurations on device
+      parameters:
+        - name: asn
+          in: path
+          type: integer
```

Comment:
uint

Reply:
There doesn't seem to be a way to specify uint. Yaml has integer data type. As per my understanding, applications will need to implement the extra logic.